folding-schemes/src/folding/nova/circuits.rs

```diff
@@ -379,7 +381,7 @@ where
         )?;
         let zero_y = NonNativeFieldVar::<C1::BaseField, C1::ScalarField>::new_constant(
             cs.clone(),
-            C1::BaseField::one(),
+            C1::BaseField::zero(),
         )?;
         (u_i.cmE.x.is_eq(&zero_x)?).conditional_enforce_equal(&Boolean::TRUE, &is_not_basecase)?;
         (u_i.cmE.y.is_eq(&zero_y)?).conditional_enforce_equal(&Boolean::TRUE, &is_not_basecase)?;
```

Comment:
small nit: maybe `zero_y` can now be removed and just use a single var containing `zero` for both `cmE.x` and `cmE.y`?